content/episode/zee-spencer-cohere-zinc-coop.md

```diff
@@ -0,0 +1,619 @@
++++
+title = "Zee Spencer on Cohere and Zinc Collective"
+date = 2021-01-07T13:00:31-08:00
+description = "Zee Spencer is a software engineer and member of Cohere, a cooperative software consultancy, and Zinc Collective, a software development collective. He talks about his journey to cooperativism, his thoughts on decision-making, fundraising (or not!), and the importance of meeting people where they're at."
+tags = []
+draft = true
+podcast_file = "https://archive.org/download/real-coop-stories-zee-spencer-on-cohere-and-zinc-collective/Real%20Coop%20Stories%20-%20Zee%20Spencer%20on%20Cohere%20and%20Zinc%20Collective.mp3"
+podcast_duration = "1:01:37"
+podcast_bytes = "59278308"
+episode_image = "/img/zee-spencer-cohere-zinc-collective.jpg"
++++
+
+## Episode notes
+
+Zee can be found on Mastodon as [@zee@wandering.shop](https://wandering.shop/@zee) and Twitter as [@zspencer](https://twitter.com/zspencer). His personal website is [zeespencer.com](https://www.zeespencer.com/). The cooperatives Zee focuses on in this interview are [Cohere](https://www.wecohere.com/) and [Zinc Collective](https://www.zinc.coop/).
+
+Bay Area organizations doing work to support mutual aid and cooperative work:
+  * [Sustainable Economies Law Center](https://www.theselc.org/)
+  * [East Oakland Collective](https://www.eastoaklandcollective.com/)
+  * [Gregory Jackson](https://www.instagram.com/melaphonik/) and [Repaired Nations](http://repairednations.org/)
+  * [East Bay Permanent Real Estate Cooperative](https://ebprec.org/)
+
+Impact and cooperative business creation and investment:
+  * [The Next Egg](https://www.thenextegg.org/)
+  * [Jenny Kassan](https://www.jennykassan.com/) and [Angels of Main Street](https://www.angelsofmainstreet.com/)
+  * [Aniya Williams](https://twitter.com/operaqueenie) and [Zebras Unite](https://zebrasunite.coop/)
+  * [Kachuwa Impact Fund](https://www.kachuwaimpactfund.com/)
```

Comment:
This list is missing Arlan's fund, Backstage Capital

Reply:
Added!